src/ontology/mondo-edit.obo

```diff
@@ -533101,8 +533097,7 @@ is_a: MONDO:0100546 {source="https://orcid.org/0000-0002-4142-7153|OMIM:620775"}
 id: MONDO:0958333
 name: thrombocytopenia 13, syndromic
 xref: OMIM:620776 {source="MONDO:equivalentTo"}
-is_a: MONDO:0100241 {source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"} ! inherited thrombocytopenia
-is_a: MONDO:0100241 {source="OMIM:620776"} ! inherited thrombocytopenia
+is_a: MONDO:0100241 {source="OMIM:620776", source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"} ! inherited thrombocytopenia
```

Comment:
should this source be written like this: `source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"` with the "|"?

Reply:
that was from the ROBOT template, this is wrong, I'll fix - good catch, thanks!

Reply:
done